packages/grid/helm/syft/templates/backend-secret.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
MAYBE: we could rename this file to syft-secret.yaml

They currently only used by the backend container.
For a normal, naming it as syft would be obvious to them as it is syft specific.